openshift/hac-dev
# Extension port

## Description
This will make hac-dev as plugin in hac-core. There is no longer AppEntry and we do not wrap components in providers since all of that should be done in hac-core.

## Type of change



## How to test or reproduce?
It should be enough just to install latest deps and run `ENVIRONMENT=prod npm run start:beta` (we have one last PR to fully support this https://github.com/openshift/hac-core/pull/35)

## Known limitations
There are currently two know limitations:
- no dry run for create resource (@vojtechszocs  @andrewballantyne are there any plans in supporting this?)
- no redux, we are working on adding redux to hac-core and will keep you updated on the process
